run_kraken.py

```diff
@@ -277,6 +277,57 @@ def litmus_scenarios(scenarios_list, config, litmus_namespaces, litmus_uninstall
     return litmus_namespaces
 
 
+# Inject the cluster shut down scenario
+def cluster_shut_down(shut_down_config, config):
+    runs = shut_down_config["runs"]
+    shut_down_duration = shut_down_config["shut_down_duration"]
+    cloud_type = shut_down_config["cloud_type"]
+    if cloud_type == "aws":
```

Comment:
Can we add in the other cloud types that have been added? 
In addition, could we add in an else case for if the cloud type is not supported and to not run/exit with an error message